.github/workflows/r-cmd-check.yaml

```diff
@@ -0,0 +1,35 @@
+on: [push, pull_request]
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/setup-r@master
+      - uses: r-lib/actions/setup-pandoc@master
+      - name: Query dependencies
+        run: |
+          install.packages(c('remotes', 'rcmdcheck', 'covr', 'rmarkdown'))
+          remotes::install_deps(dependencies = TRUE)
+          saveRDS(remotes::dev_package_deps(dependencies = TRUE), "depends.Rds", version = 2)
+        shell: Rscript {0}
+      - name: Cache R packages
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.R_LIBS_USER }}
+          key: macOS-r-3.6-${{ hashFiles('depends.Rds') }}
```

Comment:
Is this for R 3.6.x only? I wonder how caching for multiple versions of R works. For Travis, it is fairly simple: https://github.com/rstudio/DT/blob/7ebd4cc472e8d08da892a4bacd95b8ac1245ab48/.travis.yml